app/web/src/components/Navigation/ExternalNavButton.tsx

```diff
@@ -15,7 +15,7 @@ export default function ExternalNavButton({
 }: ExternalNavButtonProps) {
   const classes = useNavLinkStyles();
   return (
-    <a href={route} className={classes.link}>
+    <a href={route} target="_blank" rel="noreferrer" className={classes.link}>
```

Comment:
```suggestion
    <a href={route} target="_blank" rel="noreferrer noopener" className={classes.link}>
```
Good security practice as mentioned here - https://web.dev/external-anchors-use-rel-noopener/

Reply:
That page says noopener *or* noreferrer. O actually want to keep referrer so external pages know if we are giving them traffic.

Reply:
Lol I only skimmed and didn't see the `or` - apparently `noreferrer` implies `noopener` but it's n[ot well supported by all browsers](https://stackoverflow.com/questions/50709625/link-with-target-blank-and-rel-noopener-noreferrer-still-vulnerable), so probably still best to include them both.

Apparently it doesn't affect SEO from all the top Google results, so I think it's fine to do this.

Reply:
Or maybe only do `noopener` if we want to keep the referrer? What's the downside of not keeping that around apart from messing with other site's analytics (which doesn't sound like our problem to solve)?

Reply:
I guess it doesn't really matter, committed your suggestions